scripts/Start-EditorServices.ps1

```diff
@@ -204,203 +204,73 @@ if ((Test-ModuleAvailable "PowerShellEditorServices" -RequiredVersion $parsedVer
+    $editorServicesHost =
+        Start-EditorServicesHost `
+            -HostName $HostName `
+            -HostProfileId $HostProfileId `
+            -HostVersion $HostVersion `
+            -LogPath $LogPath `
+            -LogLevel $LogLevel `
+            -LanguageServicePort $languageServicePort `
+            -DebugServicePort $debugServicePort `
+            -BundledModulesPath $BundledModulesPath `
+            -EnableConsoleRepl:$EnableConsoleRepl.IsPresent `
+            -DebugServiceOnly:$DebugServiceOnly.IsPresent `
+            -WaitForDebugger:$WaitForDebugger.IsPresent
+
+    # TODO: Verify that the service is started
+
+    $resultDetails = @{
+        "status" = "started";
+        "channel" = "tcp";
+        "languageServicePort" = $languageServicePort;
+        "debugServicePort" = $debugServicePort;
+    };
+
+    # Notify the client that the services have started
+    WriteSessionFile $resultDetails
 }
+catch [System.Exception] {
+    $e = $_.Exception;
+    $errorString = ""
+
+    while ($e -ne $null) {
+        $errorString = $errorString + ($e.Message + "`r`n" + $e.StackTrace + "`r`n")
+        $e = $e.InnerException;
+    }
 
-# Locate available port numbers for services
-$languageServicePort = Get-AvailablePort
-$debugServicePort = Get-AvailablePort
-
-Write-Host "Starting PowerShell...`n" -ForegroundColor Blue
-
-# Create the Editor Services host
-$editorServicesHost =
-    Start-EditorServicesHost `
-        -HostName $HostName `
-        -HostProfileId $HostProfileId `
-        -HostVersion $HostVersion `
-        -LogPath $LogPath `
-        -LogLevel $LogLevel `
-        -LanguageServicePort $languageServicePort `
-        -DebugServicePort $debugServicePort `
-        -BundledModulesPath $BundledModulesPath `
-        -EnableConsoleRepl:$EnableConsoleRepl.IsPresent `
-        -DebugServiceOnly:$DebugServiceOnly.IsPresent `
-        -WaitForDebugger:$WaitForDebugger.IsPresent
-
-# TODO: Verify that the service is started
-
-$resultDetails = @{
-    "status" = "started";
-    "channel" = "tcp";
-    "languageServicePort" = $languageServicePort;
-    "debugServicePort" = $debugServicePort;
-};
-
-# Notify the client that the services have started
-WriteSessionFile $resultDetails
+    ExitWithError ("An error occurred while starting PowerShell Editor Services:`r`n`r`n" + $errorString)
+}
 
 try {
     # Wait for the host to complete execution before exiting
     $editorServicesHost.WaitForCompletion()
 }
 catch [System.Exception] {
-    $e = $_.Exception; #.InnerException;
+    $e = $_.Exception;
     $errorString = ""
 
     while ($e -ne $null) {
         $errorString = $errorString + ($e.Message + "`r`n" + $e.StackTrace + "`r`n")
         $e = $e.InnerException;
     }
 
-    ExitWithError ("Caught error while waiting for EditorServicesHost to complete:`r`n" + $errorString)
+    ExitWithError ("A terminating error occurred in PowerShell Editor Services:`r`n`r`n" + $errorString)
 }
```

Comment:
This was only temporary since the signature isn't useful after the script contents have changed.  Why did signing the startup script cause issues for PSCX?  Maybe it's an indication of some other issue that I haven't seen before